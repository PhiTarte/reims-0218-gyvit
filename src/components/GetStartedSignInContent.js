import React from "react"
import { Button, Row, Container, Col } from "reactstrap"

const GetStartedSignInContent = () => (
  <div
    style={{
      width: "max-width",
      height: "100vh",
      backgroundImage:
        "url('https://www.onecomputer.fr/wp-content/uploads/2016/01/blurred-background-81.jpg')",
      position: "relative"
    }}
  >
    <Container fluid>
      <Row>
        <Col xs="9">
          <p
            style={{
              "margin-top": "49px",
              "font-size": "16px",
              "font-weight": "normal",
              "font-style": "normal",
              "letter-spacing": "normal",
              "text-align": "left",
              color: "#3b425d",
              position: "absolute",
              "margin-left": "43vh"
            }}
          >
            Already have an account ?
          </p>
        </Col>
        <Col xs="3">
          <Button
            outline
            style={{
              width: "156px",
              height: "54px",
              borderRadius: "100px",
              border: "solid 1px #7a57d1",
              color: "#7a57d1",
              "margin-top": "30px"
            }}
          >
            Sign in
          </Button>
        </Col>
      </Row>
    </Container>
  </div>
)

export default GetStartedSignInContent
