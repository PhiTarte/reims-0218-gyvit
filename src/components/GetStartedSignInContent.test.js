import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import GetStartedSignInContent from "./GetStartedSignInContent"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<GetStartedSignInContent />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<GetStartedSignInContent />).toJSON()
  expect(tree).toMatchSnapshot()
})
