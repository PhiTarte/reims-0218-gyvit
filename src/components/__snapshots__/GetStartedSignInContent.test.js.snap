// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div
  style={
    Object {
      "backgroundImage": "url('https://www.onecomputer.fr/wp-content/uploads/2016/01/blurred-background-81.jpg')",
      "height": "100vh",
      "position": "relative",
      "width": "max-width",
    }
  }
>
  <div
    className="container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-9"
      >
        <p
          style={
            Object {
              "color": "#3b425d",
              "font-size": "16px",
              "font-style": "normal",
              "font-weight": "normal",
              "letter-spacing": "normal",
              "margin-left": "43vh",
              "margin-top": "49px",
              "position": "absolute",
              "text-align": "left",
            }
          }
        >
          Already have an account ?
        </p>
      </div>
      <div
        className="col-3"
      >
        <button
          className="btn btn-outline-secondary"
          onClick={[Function]}
          style={
            Object {
              "border": "solid 1px #7a57d1",
              "borderRadius": "100px",
              "color": "#7a57d1",
              "height": "54px",
              "margin-top": "30px",
              "width": "156px",
            }
          }
          type={undefined}
        >
          Sign in
        </button>
      </div>
    </div>
  </div>
</div>
`;
